asr_deepspeech/README.md

```diff
@@ -4,7 +4,7 @@ This service is concerned with converting speech input to text output. It can re
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```

Comment:
openvino/ubuntu18_runtime:2021.3 is used

Reply:
https://github.com/intel-innersource/applications.ai.conversational-ai.chatbot/blob/master/dockerfiles/deepspeech8_asr.dockerfile

Reply:
fixed in cabf1272c5240e39836103f5147554ca8ea86acb